spec/classes/draggableSpec.js

```diff
@@ -0,0 +1,234 @@
+describe('Draggable', function () {
+    'use strict';
+
+    beforeEach(function () {
+        module('pmImageEditor');
+    });
+
+    describe('factory', function() {
+        var factory,
+            element,
+            event,

```

Comment:
This is actually mock for event (which is used in real controller). I just simplified to use object with coordinates instead of real event. So name looks logical for me.